src/transformStateWithClones.js

```diff
@@ -7,7 +7,48 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const cloneState = [];
+
+  let stateCopy = {
+    ...state,
+  };
+
+  const addAction = 'addProperties';
+  const removeAction = 'removeProperties';
+  const clearAction = 'clear';
```

Comment:
it's better to use uppercase naming once you're setting the constants
```suggestion
  const ADD_ACTION = 'addProperties';
  //...
```